courses/CONTRIBUTING.md

```diff
@@ -1,20 +1,30 @@
 We realise that the initial content we created is just a starting point and our hope is that the community can help in the journey refining and extending the contents.
 
-As a contributor, you represent that the content you submit is not plagiarised. By submitting the content, you (and, if applicable, your employer) are licensing the submitted content to LinkedIn and the open source community subject to the Creative Commons Attribution 4.0 International Public License.
+### Reporting issues
+
+We use GitHub issues to track bug reports, feature requests, and submitting pull requests.
+
+If you find a bug:
+
+1. Use the GitHub issue search to check whether the bug has already been reported.
 
-*Repository URL*: [https://github.com/linkedin/school-of-sre](https://github.com/linkedin/school-of-sre)
+1. If the issue has been fixed, try to reproduce the issue using the latest master branch of the repository.
+
+1. If the issue still reproduces or has not yet been reported, try to isolate the problem before opening an issue.
```

Comment:
numbering is incorrect

Reply:
This is correct markdown syntax, see https://www.markdownguide.org/basic-syntax/#ordered-lists